src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationSideTests.java

```diff
@@ -180,21 +180,11 @@ public void MercuryNavigationSideTest_002_SearchInvalidSuggestion_Cancel() {
 
     searchObject.typeInSearchField(SEARCH_FAIL);
 
-    result = searchObject.isSorryInfoDisplayed();
+    result = !searchObject.isSuggestionListDisplayed();
     PageObjectLogging.log(
-        "Sorry message",
-        "is displayed",
-        "is hidden",
-        result
-    );
-
-    searchObject.waitMilliseconds(5000, "Wait for message to disappear");
-
-    result = searchObject.isSorryInfoDisplayed();
-    PageObjectLogging.log(
-        "Sorry message",
-        "is displayed",
-        "is hidden",
+        "Search suggestions",
+        "are hidden",
+        "are loaded",
```

Comment:
This test only logs its results, I wonder if the test isn't phantom - never fails